src/sentry/utils/committers.py

```diff
@@ -130,6 +130,34 @@ def _get_committers(annotated_frames, commits):
     return user_dicts
 
 
+def get_previous_releases(project, start_version, limit=5):
```

Comment:
ideally we'd have a 'start_release' here rather than a 'start_version', but Group.get_first_release is returning a string instead of the release object itself